content/no-code.md

```diff
@@ -88,16 +87,13 @@ This page briefly introduces a standard to help you make your no-code website to
 
 ## Why Accessibility Matters
 
-People with disabilities use your no-code tool to create and consume websites.
+Your no-code tool users include people with disabilities. Your tool needs to be accessible to disabled content creators.
 
-Your tool will be more appealing and more widely used if it is accessible to website creators and consumers.
+The websites that your tool outputs also need to be accessible to disabled website users.
```

Comment:
```suggestion
The websites and widgets that your tool outputs also need to be accessible to disabled website users.
```

These tools may also output components or widgets. These may or may not be part of the whole website. Is it worth for uss to make that difference?

Reply:
I don't know. Perhaps leave it simply just websites for now and leave an open issue to revisit when we can get more input. Maybe open a new GitHub issue and ask EOWG to comment?

Reply:
Thanks. I'll add for now and open separate issue in case we decide to go back.